src/App.css

```diff
@@ -46,6 +46,8 @@
   overflow: hidden;
   background-color: #333;
   font-family: Arial;
+  list-style-type: none;
```

Comment:
I would recommend 'mobile-first' design. When i view this in any view other than desktop, the responsiveness needs work